arango_datasets/datasets.py

```diff
@@ -30,8 +31,9 @@ def __init__(
     ):
         self.metadata_file: str = metadata_file
         self.metadata_contents: Dict[str, Any]
-        self.batch_size = batch_size
+        self.batch_size = batch_size or 50
```

Comment:
Would it make sense to introduce this as a default in the `__init__` arguments? changing the type of `batch_size` from `Optional[int]` to `int` with a default of `50`?